core/controller/src/main/scala/whisk/core/controller/Rules.scala

```diff
@@ -152,13 +152,13 @@ trait WhiskRulesApi extends WhiskCollectionAPI with ReferencedEntities {
             case Failure(t) =>
               t match {
                 case _: DocumentConflictException =>
-                  logging.info(this, s"[POST] rule update conflict")
+                  logging.warn(this, s"[POST] rule update conflict")
                   terminate(Conflict, conflictMessage)
                 case IgnoredRuleActivation(ok) =>
-                  logging.info(this, s"[POST] rule update ignored")
+                  logging.warn(this, s"[POST] rule update ignored")
                   if (ok) complete(OK) else terminate(Conflict)
                 case _: NoDocumentException =>
-                  logging.info(this, s"[POST] the trigger attached to the rule doesn't exist")
+                  logging.warn(this, s"[POST] the trigger attached to the rule doesn't exist")
```

Comment:
Here also the use of `warn` is concerning. I think these should be reserved to system invariant warnings not errors that can be caused by the client. This is why these are at info level. Suggest `debug` instead.